en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,77 @@
-# PX4 Continuous Integration
+# PX4 Continuous Integration / Automation
 
-PX4 builds and testing are performed using GitHub actions and a Jenkins server.
\ No newline at end of file
+PX4 has multiple automated CI build that runs to keep different systems in sync. This document describes how they are configured & connected, to hopefully clear-up confusion in how everything is connected!
+
+:::note
+CI scripts are subject to change, and the links below may get broken in the future. If that happens, please contribute the fix for the documentation!
+:::
```

Comment:
FYI You could add this to every bit of docs in the library. Everything is subject to change, and if someone finds something wrong we hope they'll fix it. Why is this special? Deleting.